src/popup/fetch.ts

```diff
@@ -26,6 +26,21 @@ function roundToNextHour(date: Date): Date {
 	return date;
 }
 
+function reformatDate(date_str: string): string {
```

Comment:
A super pedantic thing, but would you mind please changing this to camelCase? `dateString` would be amazing 🙂

Reply:
Done!

Reply:
Thank you 😃